vault/resource_database_secret_backend_role_test.go

```diff
@@ -164,5 +193,20 @@ resource "vault_database_secret_backend_role" "test" {
   max_ttl = 3600
   creation_statements = ["SELECT 1;"]
 }
+
+resource "vault_database_secret_backend_role" "role" {
+  backend             = vault_mount.db.path
+  name                = "dev"
+  db_name             = vault_database_secret_backend_connection.test.name
+  creation_statements = ["{'database_name': '$external', 'x509Type': 'CUSTOMER', 'roles': [{'databaseName':'dbName','roleName':'readWrite'}]}"]
+  credential_config = {
```

Comment:
Same comment as above in regards to `credential_type`

Reply:
Should be fixed now